.github/workflows/ci.yml

```
@@ -327,3 +327,16 @@ jobs:
           prerelease: false
           files: |
             package-*/*
+            
+  winget-publish:
+    needs: release
+    if: (runner.os == 'Windows') && contains(github.ref, 'tags/v')
+    runs-on: windows-latest
+    steps:
+      - name: Submit package to Windows Package Manager Community Repository
+        uses: ./.github/actions/winget-publish
+        with:
+          release-version: ${{env.DESKFLOW_VERSION}}
+          package-id: 'Deskflow.Deskflow(64-bit)'
```

```suggestion
          package-id: 'Deskflow.Deskflow'
```

I wonder if 64-bit is redundant, since we don't offer a 32-bit MSI.